rdt/transformers/base.py

```diff
@@ -66,7 +66,7 @@ def get_output_types(self):
 
         Returns:
             dict:
-                Mapping from the transformed column names to the produced data types.
+                Mapping from the transformed column names to the produced sdtypes.
```

Comment:
@npatki Should the `get_input_type` and `get_output_type` method change to be `get_input_sdtype` and `get_output_sdtypes` respectively?

Reply:
Sure that makes sense.